src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -637,13 +636,15 @@ export default class DefaultAudioVideoController
         poorConnectionCount,
         startTimeMs,
         iceGatheringDurationMs,
+        selfAttendeePresentDurationMs,
       } = this.meetingSessionContext;
       const attributes: AudioVideoEventAttributes = {
         maxVideoTileCount: this.meetingSessionContext.maxVideoTileCount,
         meetingDurationMs: startTimeMs === null ? 0 : Math.round(Date.now() - startTimeMs),
         meetingStatus: MeetingSessionStatusCode[status.statusCode()],
         signalingOpenDurationMs,
         iceGatheringDurationMs,
+        selfAttendeePresentDurationMs,
```

Comment:
Nice! Just do not include this attribute for the `meetingStartFailed` event. The `meetingStartFailed` event means that builders do not even receive the `audioVideoDidStart`. `selfAttendeePresentDurationMs` adds confusion.

Reply:
Nice catch!!
Added `delete attributes.selfAttendeePresentDurationMs;` before publishing the `meetingStartFailed` event